app/assets/index.html

```diff
@@ -20,5 +20,6 @@
 </div>
 
 <script src="scripts/main.js"></script>
+<script>require('app/scripts/main');</script>
```

Comment:
`public/scripts/main.js` is also loaded in Karma, but Karma isn't initializing the application like we would when we are present it in the browser (in the `main` module); rather, Karma only wants the module definitions so it can require them via "*.test.js" files. Building `public/scripts/main.js` without `autoRequire` makes it suitable for use in both contexts, the caveat being that we need to initialize the application this way in the browser.

There are two alternatives:

- CSP-friendly, config-friendly, performance-unfriendly: Put the `main` module into another script file.
- CSP-friendly, config-unfriendly, performance-friendly: Continue building `public/scripts/main.js` with `main` `autoRequire`'d, and also build `public/scripts/modules.js` in development for use in Karma.